pulp_rpm/app/tasks/publishing.py

```diff
@@ -418,6 +428,14 @@ def create_repomd_xml(
     pkg_to_hash = {}
     for ca in contentartifact_qs.iterator():
         pkgid = None
+        if ca.content.rpm_package.checksum_type not in settings.ALLOWED_CONTENT_CHECKSUMS:
+            raise ValueError(
+                "Package {} does not contain allowed chucksum type, thus can't be published. "
```

Comment:
s/chucksum/checksum/

Reply:
Since we're returning the same error message from multiple places, it might be good to make a custom error type, or store it in a variable so that it can be reused.  Like ```ALLOWED_CHECKSUM_ERROR_MSG```

Reply:
Only last lines of error messages are the same as pointing to same setting value.